.github/workflows/code_quality_checks.yml

```diff
@@ -48,6 +51,17 @@ jobs:
             - name: Run Vitest
               run: npm run test:unit
 
+            - name: Setup LCOV
+              uses: hrishikesh-kadam/setup-lcov@v1
+
+            - name: Report code coverage
+              uses: zgosalvez/github-actions-report-lcov@v3
+              with:
+                  coverage-files: coverage/lcov.info
+                  artifact-name: code-coverage-report
+                  github-token: ${{ secrets.GITHUB_TOKEN }}
+                  update-comment: true
+
     e2e-test:
```

Comment:
Maybe add to this job the upload of the playwright report (https://playwright.dev/docs/ci).

Reply:
Good catch, I added it.

Reply:
Perfect.